README.md

```diff
@@ -44,11 +44,6 @@ Mixing packages with other distributions is not supported and can create securit
 
 ## If Wolfi is missing a package you require
 
-Wolfi is not currently intended to be a general purpose desktop operating system. Our priority is to provide packages
-that enable containerized and embedded system workflows. Please keep this in mind when proposing adding packages to
-Wolfi. Also note that some packages may not be appropriately licensed for inclusion.  FSF or OSI approved [licenses](https://spdx.org/licenses/) are ideal.
+Wolfi is the core foundation of Chainguard OS and Chainguard's Starter container images.
```

Comment:
Chainguard's Starter container images

is that really a product / feature name and hence you capitalize "Starter" .. I think this is bad and we should say something liek

the free starter images available from the Chainguard Containers catalog.